.github/workflows/testing_multiversion.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         with:
           python-version: ${{matrix.python-version}}
           poetry-version: 1.2.2
+          install-args: --all-extras
```

Comment:
Testing should be done with the minimal installation, to make sure that this works.

Reply:
In future, there might be other optional dependency groups that don't impact general functionality but e.g. specific loaders. There is a possibility of including dask-jobqueue to schedule maxwell nodes through slurm for flashloader. Now this dependency wouldn't strictly be necessary for other modules. 
Or e.g. integrating multidim visualization tools such as Napari. 
In such cases, we should also allow for testing whole suite (maybe another workflow).

Reply:
Then we should as you suggest provide separate tests for these, I suggest.